crates/pessimistic-proof/Cargo.toml

```diff
@@ -6,24 +6,16 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-base64 = "0.22.0"
 reth-primitives = { git = "https://github.com/sp1-patches/reth", default-features = false, branch = "sp1-reth" }
 serde = { version = "1", features = ["derive"] }
 serde_json = { version = "1", features = ["arbitrary_precision"] }
 serde_with = { version = "3" }
-thiserror.workspace = true
+thiserror = "1.0.58"
```

Comment:
Note: this is to fix the compilation of the proof ELF binary. When compiling the proof, the crate is not technically a part of the top-level workspace, so specifying a workspace dependency sadly does not work.

Reply:
Which command fails ? The `cargo xtask elf` ?

Reply:
> Which command fails ? The `cargo xtask elf` ?

Yes

Reply:
It's weird, as for me and for the CI it is working

Reply:
Hmm must have been something else then, works for me too now. Sorry for the confusion.